### DISCUSSION TITLE
What is the purpose of Sidebery "sync", and how to use it?

### DISCUSSION THREAD

**Discussion by wsalopek** [mbnuqw/sidebery#1030] [General]
Does "sync" automatically send/receive data to/from two different computers, or the cloud, etc?  It doesn't look like it, but if it's not syncing with another device/cloud, what does syncing actually do?

**Comment by mbnuqw** [comment]
Sync uploads selected data to Firefox sync-server (via [sync API](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync)) and on another profile it loads that data, so you can manually check what data you have, from what device and use the needed.
